pypi/ragas_once/setup.py

```diff
@@ -0,0 +1,29 @@
+# make a setup.py for ragacli package
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="raga-once",
```

Comment:
ragas-once or ragas_once

Reply:
The pypi package name is raga-once, while python subdir name is ragas_once since python requires it. (e.g. python can only read `import ragas_once.cli as pkg`, but not `import ragas-once.cli as pkg`